uropa/visualization.py

```diff
@@ -26,29 +31,58 @@ def distribution_plot(table, var, kind):
     """
     pass
 
-
-def count_plot(table, var, kind):
+def count_plot(table, var, kind, title=None, path=None, dpi="figure"):
     """
     Count and plot the occurence of the selected categorical variable.
     Either shown as a pie chart or bar plot.
-
     Parameters
     ----------
     table : pd.DataFrame
         Pandas dataframe containing the data.
-    var : <datatype>, <default value>
-        <param description>
-    kind : <datatype>, <default value>
-        <param description>
-    TODO add more parameters
-
+    var : String value
+        String value naming column along which to group peaks
+    kind : String value
+        String value naming plot type (pie, bar)
+    title : String value, default=None
+        String value for title of plot. If None plot has no title.
+    path : String value, default=None
+        String value with path to save plot at. If None plot is not saved.
+    dpi : Float value, default="figure"
+        Float value with DPI to save plot with. If "figure" the figure's DPI is used.
+    TODO test function and add par plot option
     Returns
     -------
     <datatype> :
         <return description>
     TODO should return the plotting object
     """
-    pass
+    valid_columns = table.keys() # List of valid column names from input table

```

Comment:
Please add a check if the given name in `var` is a valid column name.

Reply:
There is no need to save the column names in a variable if you are using it only once.
Also, while this works the same you usually get column names with `table.columns`. Please change that as it is a bit more clear what it does.

Reply:
I replaced table.keys() with table.columns and skipped saving the result in a variable.